jas/sast/sastscanner.go

```diff
@@ -97,6 +104,7 @@ func (ssm *SastScanManager) createConfigFile(module jfrogappsconfig.Module, excl
 				Roots:           roots,
 				Language:        sastScanner.Language,
 				ExcludedRules:   sastScanner.ExcludedRules,
+				SastParameters:  sastParameters{SignedDescriptions: signedDescriptions},
```

Comment:
I'm thinking whether we should add "`SastParameters`" under "`SastScanManager`" too